src/@next/components/atoms/Tile/test.tsx

```diff
@@ -55,6 +56,29 @@ describe("<Tile />", () => {
     );
   });
 
+  it("should display only element if no footer and header elements passed", () => {
```

Comment:
only one element?

Reply:
Yes :) it should be only one element. Fixed